From 2e422109af7094d2787660f7841e7c2ab10bbcc8 Mon Sep 17 00:00:00 2001
From: Pavel Vishnyakov <djhumster@gmail.com>
Date: Thu, 17 Dec 2015 18:00:37 +0300
Subject: [PATCH] Update conf.traceroute

More correct way to match domains with any length.
Can be checked her https://regex101.com/r/dI3jQ3/1
---
 conf.traceroute | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/conf.traceroute b/conf.traceroute
index b6501a2..983a7cd 100644
--- a/conf.traceroute
+++ b/conf.traceroute
@@ -1,5 +1,5 @@
 # hostname
-regexp=\s\w+[\w\-\.]+\w+
+regexp=\w+[\w\-\.]+\.[a-z]+
 colours=bold yellow
 count=once
 -
